frontend/assets/meta.user/res/js/fields/URL.jsx

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright 2007-2021 Charles du Jeu - Abstrium SAS <team (at) pyd.io>
+ * This file is part of Pydio.
+ *
+ * Pydio is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Pydio is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Pydio.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * The latest code can be found at <https://pydio.com>.
+ */
+/**
+ * @type {import('react')} React
+ */
+import React, { Fragment, useCallback, useMemo, useRef, useEffect, useState } from 'react'
+import Pydio from 'pydio'
+import asMetaField from "../hoc/asMetaField";
+import asMetaForm from "../hoc/asMetaForm";
+const { ModernTextField, ThemedModernStyles } = Pydio.requireLib('hoc');
+import { muiThemeable } from 'material-ui/styles'
+import { FontIcon } from 'material-ui'
+import { debounce } from 'lodash'
+
+/**
+ * @param {{fontSize?: number}} props
+ * @returns {React.ReactElement}
+ */
+const URLIcon = ({ fontSize }) =>
+    <FontIcon
+        data-testid="open-in-new-icon"
+        className="mdi mdi-open-in-new"
+        style={{ fontSize }} />
+
+/**
+ * @param {{
+ *   fontSize?: number,
+ *   url: string,
+ *   displayText?: string,
+ *   children?: React.ReactNode
+ * }} props
+ *
+ *  @returns {React.ReactElement}
+ */
+const URLLinkIcon = ({ fontSize, url, displayText, children }) => {
+    const hasMinimalValidity = /^https?:\/\//i.test(url);
+    if (!hasMinimalValidity) {
+        return null;
+    }
+
+    const labelText = displayText || children || url;
+
+    return (
+        <a href={url}
+            target="_blank"
+            aria-label={`Open ${labelText} in a new tab`}
+            rel="noopener noreferrer"
+            onClick={(e) => {
+                e.stopPropagation();
+            }}
+            style={{
+                color: 'var(--md-sys-color-primary)',
+                textDecoration: 'none'
+            }}
+        >
+            <URLIcon fontSize={fontSize} />
+            {children}
+        </a>
+    )
+
+}
+
+/**
+ * @param {{getRealValue: () => string}} props
+ * @returns {React.ReactElement}
+ */
+const URLFieldBase = ({ getRealValue }) => {
+    const value = getRealValue();
```

Comment:
Nitpick May be simplified 
```suggestion
    const url = getRealValue();
```

Reply:
Changed since now I use a sanitized URL